src/Deployment/CliRunner.php

```diff
@@ -140,6 +140,9 @@ private function createDeployer($config)
 		$deployment->runAfterUpload = self::toArray($config['afterupload'], TRUE);
 		$deployment->runAfter = self::toArray($config['after'], TRUE);
 		$deployment->testMode = !empty($config['test']) || $this->mode === 'test';
+		$deployment->filePerms =  empty($config['filePerms']) ? '' : $config['fileperms'];
```

Comment:
Yes, it is typo. Maybe config parameters should be the same in *.php and .ini files including capitalization?  (i.e.: afterupload vs afterUpload)